aptos-move/framework/aptos-experimental/sources/trading/order_book/order_book_types.move

```diff
@@ -29,31 +26,45 @@ module aptos_experimental::order_book_types {
         account_order_id: u64
     }
 
+    // Internal type representing order in which trades are placed. Unique per instance of AscendingIdGenerator.
     struct UniqueIdxType has store, copy, drop {
         idx: u256
```

Comment:
Let's make this u128 as that will be the width of monotonically increasing counter?